src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -27,78 +27,82 @@
 				:value.sync="searchText"
 				class="conversations-search"
 				:class="{'conversations-search--expanded': isFocused}"
-				:is-searching="isSearching"
+				:is-focused="isFocused"
 				@focus="setIsFocused"
 				@blur="setIsFocused"
+				@trailing-blur="setIsFocused"
 				@input="debounceFetchSearchResults"
 				@submit="onInputEnter"
 				@keydown.enter.native="handleEnter"
 				@abort-search="abortSearch" />
 
-			<!-- Filters -->
-			<div class="filters"
-				:class="{'hidden-visually': isFocused}">
-				<NcActions class="filter-actions"
-					:primary="isFiltered !== null">
-					<template #icon>
-						<FilterIcon :size="15" />
-					</template>
-					<NcActionButton close-after-click
-						class="filter-actions__button"
-						:class="{'filter-actions__button--active': isFiltered === 'mentions'}"
-						@click="handleFilter('mentions')">
-						<template #icon>
-							<AtIcon :size="20" />
-						</template>
-						{{ t('spreed','Filter unread mentions') }}
-					</NcActionButton>
-
-					<NcActionButton close-after-click
-						class="filter-actions__button"
-						:class="{'filter-actions__button--active': isFiltered === 'unread'}"
-						@click="handleFilter('unread')">
-						<template #icon>
-							<MessageBadge :size="20" />
-						</template>
-						{{ t('spreed','Filter unread messages') }}
-					</NcActionButton>
-
-					<NcActionButton v-if="isFiltered"
-						close-after-click
-						class="filter-actions__clearbutton"
-						@click="handleFilter(null)">
+			<transition-group name="radial-reveal">
+				<!-- Filters -->
+				<div v-show="!isFocused" key="filters" class="filters">
```

Comment:
```suggestion
				<div v-show="!isSearching" key="filters" class="filters">
```

Non-blocking: I'd replace this parameter for better handling of keyboard navigation:

`isFocused`:

https://github.com/nextcloud/spreed/assets/93392545/35cbe42a-d215-4132-9a37-e26d6dd7272d

`isSearching`:

https://github.com/nextcloud/spreed/assets/93392545/1a8c929f-9586-402f-9ced-703375f6455c

:warning: Fancy animation won't work with this configuration, so would be wise to discuss with design team

Also outline radius for trailing button should be round, like here:
```
:deep(.input-field__input) {
	border-radius: var(--border-radius-pill);
}
```